src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -60,7 +60,7 @@ type Props = {
 
 export default function LineNumberTooltipWrapper(props: Props) {
   const { value } = useFeature("hitCounts");
-  return value ? null : <LineNumberTooltip {...props} />;
+  return (false && value) ? null : <LineNumberTooltip {...props} />;
```

Comment:
I don't get it.

Reply:
oh that wasnt supposed to be there. meant to just remove the feature flag.